### DISCUSSION TITLE
Monthly meeting: 5. July 2022

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#1744] [Show and tell]
## Topics

- semantic releases (husky …)
  - do we want it?
  - how do we handle work of outside collaborators?
  - how do we "enforce" commit naming conventions?
  - manually triggered (pipeline for) releases might be more favorable than automatic release on merge
  - do we want to put access tokens to github?
  - discussion to be continued
- dev setup https://github.com/geostyler/geostyler/issues/1790
  - yes, let's do this
- error boundaries
  - do we still want them?
    - @KaiVolland -> No
  - **needs clarification** -> next monthly
- CardLayout as additional Layout or as replacement for for non-compact?
  - replace non-compact layout
- Windows dev setup https://github.com/geostyler/geostyler/issues/1769
- release version 9
  - after-codesprint-release
  - browser build needs to be fixed first https://github.com/geostyler/geostyler/issues/1804
- use issue forms https://github.com/geostyler/geostyler/issues/1805
- TODO @dnlkoch remove geoserver plugin and point links to dedicated repo in geoserver documentation
- osgeolive https://github.com/geostyler/geostyler/issues/1776

## v10 features / RoadMap
- Definetly
  - Expressions (UI, SLD and OL parser)
- Nice to have
  - UnsupportedProperties Feedback directly on the EditorFields
  - Semantic-Releases :arrow_up:

TODO for next monthly, directly invite code sprint participants!!

Next monthly: https://github.com/geostyler/geostyler/discussions/1806

**Comment by jansule** [comment]
Link to Session: https://meet.jit.si/GeoStylerCS
